examples/internal/proto/examplepb/unannotated_echo_service.swagger.json

```diff
@@ -47,8 +47,6 @@
     },
     "/v1/example/echo/{id}/{num}": {
       "get": {
-        "summary": "Echo method receives a simple message and returns it.",
-        "description": "The message posted as the id parameter will also be\nreturned.",
```

Comment:
This change is incorrect, so we need to figure out how we can fix your case and still keep this.